.gitmodules

```
@@ -1,6 +1,3 @@
 [submodule "content/Main"]
```

Not sure that it matters, but:
```suggestion
[submodule "content/main"]
```